simpleLooping/test_normalizeString.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from normalizeString import normalizeString
+
+class TestNormalizeString(unittest.TestCase):
+    def test_ListStringWithSpaceInTheFront(self):
+        self.assertEqual(normalizeString(" Hello World!"), "Hello World!")
+
+    def test_listStringWithSpaceInTheEnd(self):
+        self.assertEqual(normalizeString("Hello World!"  ), "Hello World!")
```

Comment:
Are you sure there is space at the end of your string? Those 2 below are not equal.

("Hello World!"  )
("Hello World!  ")